models/dave_object_models/package.xml

```diff
@@ -3,7 +3,7 @@
   <name>dave_object_models</name>
   <version>3.1.1</version>
   <description>The dave_object_models package</description>
-
+  <author email="mabel@openrobotics.org">Mabel Zhang</author>
```

Comment:
I can't claim authorship here. I only added 1 file to the package very recently - probably why last modified shows me.

Reply:
Removed author tag with 7658a50